src/mailer/templates/account-closure-notification.ts

```diff
@@ -0,0 +1,67 @@
+import { Dinero } from 'dinero.js';
+import MailContent from './mail-content';
+import { signatureDutch, signatureEnglish } from './signature';
+import MailTemplate, { Language, MailLanguageMap } from './mail-template';
+
+interface AccountClosureNotificationOptions {
+  name: string;
+  balance: Dinero;
+}
+
+const accountClosureNotificationDutch = new MailContent<AccountClosureNotificationOptions>({
+  getHTML: (context) => `
+<p>Beste ${context.name},</p>
+
+<p>Wij willen u informeren dat uw account bij SudoSOS is gesloten. Op het moment van sluiting was het saldo van uw account:<br>
```

Comment:
Perhaps we want to include why this happened, although it reduces the reusability of these notifications. I am afraid a lot of people are going to contact bacpm while what they should do is click the link in their email inbox (which is valid [for 31 days after the membership expires](https://github.com/GEWIS/gewisdb/blob/1d9ddf7a924e9587e2545d7350d2132ec34a7293/module/Database/src/Model/RenewalLink.php#L82) to cover these cases)

Reply:
Eens met Rink. Ik denk dat we hier wel een apart geval van kunnen maken. Ook omdat je mensen wilt laten weten dat ze hun account kunnen laten omzetten naar een `LOCAL_USER` als ze geen GEWIS lid meer willen blijven, maar nog wel SudoSOS willen gebruiken.

Reply:
Changed the wording.